deploy/sdk/src/dynamo/sdk/cli/serve.py

```diff
@@ -85,16 +85,22 @@ def serve(
         False,
         help="Save a snapshot of your service state to a file that allows planner to edit your deployment configuration",
     ),
+    target: str = typer.Option(
```

Comment:
Can we set this to match a set of enums, so that validation can be done immediately?